code/src/js components/Button3.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import "css components/button3.css";
+
+const goback = () => {
+    document.location.href = "";
+};
+
+export const Button3 = () => {
+  
+  return (
+    <>
+    <button className="resetbutton" onClick={goback}type="reseting">reset</button>
```

Comment:
I like that you uses onClick here instead of adding an a-element. You know your Javascript-skills! ;)